src/main/java/io/quarkus/activity/github/GitHubService.java

```diff
@@ -41,7 +41,7 @@
 
     private final String token;
 
-    @ConfigProperty(name = "activity.logins", defaultValue = "rsvoboda,mjurc,pjgg,Sgitario,kshpak,jsmrcka,fedinskiy")
```

Comment:
I wouldn't change this yet, let's have Jose and Kyrylo still tracked for historical data in monthly stats mainly